tests/behat/features/capabilities/post/post-modal-comments.feature

```diff
@@ -0,0 +1,33 @@
+@api @post @stability @perfect @critical @YANG-4830 @database @stability-2 @post-modal-comments
+Feature: Comment on a Post with an Image
+  Benefit: In order to give my opinion on a post with an image
+  Role: As a LU
+  Goal/desire: I want to comment on a post with an image
+
+  Scenario: Successfully create a comment on a post with an image in the modal window
+
+    Given I enable the module "social_ajax_comments"
```

Comment:
@OH17L This behat test looks good, but since we made changes in `social_post_album` module in this PR, I think we need to write a scenario when this module is enabled. Could you enable it here or maybe even better write a separate similar scenario, but with social_post_album enabled.

Reply:
As you can see [here](https://github.com/goalgorilla/open_social/blob/10.1.x/modules/social_features/social_core/social_core.install#L1291) this module is enabled by default.